src/SkeletalTrapezoidation.h

```diff
@@ -188,7 +187,7 @@ class SkeletalTrapezoidation
 
     // ^ init | v transitioning
 
-    void setMarking(); //!< set the is_marked flag for each edge based on the transitioning_angle
```

Comment:
Pretty sure the `//<!` style comments show up elsewhere in the engine as well, are you sure we should replace them with plain `//`?

Reply:
No, but i'm also not sure as to why the <! is in there in the first place. My IDE (Kdevelop) just ignores it.

Reply:
Ok, fair enough!

Reply:
It's a Doxygen thing for a single-line documentation entry.
See: http://doxygen.nl/manual/docblocks.html#memberdoc